oneoffs/heatmap.py

```diff
@@ -0,0 +1,86 @@
+"""
+Used to plot a heatmap of the policy and value networks.
+Check FLAGS for default values for what models to load.
+
+Usage:
+python heatmap.py
+
+"""
+import sys; sys.path.insert(0, '.')
+
+import itertools
+import os
+
+import numpy as np
+import tensorflow as tf
+from tqdm import tqdm
+
+import go
+import oneoff_utils
+
+
+tf.app.flags.DEFINE_string(
+    "sgf_dir", "sgf/baduk_db/", "sgf database")
```

Comment:
This is 95% not where you want to point but I don't have a better path to default to

Reply:
this is fine.  The docstring should be better though; "path to directories containing SGFs" or something.